pkgs/top-level/all-packages.nix

```
@@ -36397,6 +36397,8 @@ with pkgs;
 
   vpcs = callPackage ../applications/virtualization/vpcs { };
 
+  vsc-leetcode-cli = callPackage ../applications/misc/vsc-leetcode-cli { };
```

```suggestion
  vsc-leetcode-cli = callPackage ../tools/misc/vsc-leetcode-cli { };
```
because it's a CLI tool